.editorconfig

```diff
@@ -4,7 +4,7 @@ indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
-end_of_line = lf
+end_of_line = crlf
```

Comment:
Why this?

Even if here, it is an unrelated change. I really don't want this included in this PR:

Reply:
Unintended, forgot to push the commit that reverts it.